src/main/java/mat/client/cqlworkspace/attributes/FhirAttributeSelectionEvent.java

```diff
@@ -21,31 +25,24 @@ protected void dispatch(Handler handler) {
         handler.onAttributeSelectionChanged(this);
     }
 
-    public interface Handler extends EventHandler {
-        void onAttributeSelectionChanged(FhirAttributeSelectionEvent event);
-    }
-
     public boolean isSelected() {
         return selected;
     }
 
-    public void setSelected(boolean selected) {
-        this.selected = selected;
-    }
-
-    public FhirAttribute getAttribute() {
-        return attribute;
-    }
-
-    public void setAttribute(FhirAttribute attribute) {
-        this.attribute = attribute;
+    public String getAttributeId() {
+        return attributeId;
     }
 
     @Override
     public String toString() {
         return "FhirAttributeSelectionEvent{" +
                 "selected=" + selected +
```

Comment:
I would probably use `@ToString` in Lombok for this so you don't have to change it when you add member bars.

Reply:
@cday-semanticbits  that's the client side code that is transpiled to JavaScript. I'm not sure if lombook annotations  will work with GWT out of the box.
Do you know how to make it work?
There is a claim it can be done https://projectlombok.org/setup/gwt

Reply:
ah right forgot about it having to work with gwt ... probably not worth figuring out then.